src/components/Send.jsx

```diff
@@ -290,121 +290,110 @@ export default function Send({ makerRunning, coinjoinInProcess }) {
           </rb.Col>
         </rb.Row>
```

Comment:
Should we also change the markup for the loading indicator?
From
```jsx
<rb.Row className="justify-content-center">
  <rb.Col className="flex-grow-0">
    <div className="d-flex justify-content-center align-items-center">
      <rb.Spinner as="span" animation="border" size="sm" role="status" aria-hidden="true" className="me-2" />
      Loading
    </div>
  </rb.Col>
</rb.Row>
```

to:
```jsx
<div className="d-flex justify-content-center align-items-center">
  <rb.Spinner as="span" animation="border" size="sm" role="status" aria-hidden="true" className="me-2" />
  Loading
</div>
```


(Does GitHub not allow suggestions of unchanged code in a review?)

Reply:
Sure Row/Col not needed anymore! 👍

Reply:
> (Does GitHub not allow suggestions of unchanged code in a review?)

Nope, only changed lines. 😉